extensions/interactions/MusicNotesInput/static/css/musicNotesInput.css

```diff
@@ -8,6 +8,7 @@
   margin: 0 0 0 11.0784%;
   width: 87.34525%;
   height: 43%;
+  visibility: hidden;
```

Comment:
Ah -- I think you can keep the earlier comment, it is useful. The thing I was objecting to was the commented-out "display: none;" bit, which made it unclear whether that was supposed to be part of this CSS rule or not.